db/schema/transactionPerson.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose, { Schema } from 'mongoose';
+
+import transaction from './transaction';
+import amountDetail from './amountDetail';
+import item from './item';
+
+const transactionPersonSchema: Schema = new mongoose.Schema({
+  id: String,
+  person: String,
+  transaction: transaction,
+  amountDetails: amountDetail,
+  items: [item],
+  status: {
+    type: String,
+    enum: ['COMPLETED', 'ONGOING', 'DELETED'],
```

Comment:
Should this enum have a schema or be a const somewhere since we're using the same status in transaction schema as well?